extensions/geode-modules/src/main/java/org/apache/geode/modules/util/CreateRegionFunction.java

```diff
@@ -179,17 +176,14 @@ public boolean hasResult() {
     return true;
   }
 
-  @Override
-  public void init(Properties properties) {}
-
   private RegionStatus createRegion(RegionConfiguration configuration) {
     // Get a distributed lock
     DistributedMemberLock dml = getDistributedLock();
     if (this.cache.getLogger().fineEnabled()) {
       this.cache.getLogger().fine(this + ": Attempting to lock " + dml);
     }
-    long start = 0, end = 0;
-    RegionStatus status = null;
+    long start = 0, end;
```

Comment:
won't start also default to 0?

Reply:
Not initialising the `start` variable to `0` makes the compilation fail with `CreateRegionFunction.java:196: error: variable start might not have been initialized`.